sources/api/netdog/build.rs

```diff
@@ -14,6 +25,19 @@ fn main() {
         }
     };
     variant.emit_cfgs();
+    emit_net_backend_cfgs();
 
     generate_readme::from_main().unwrap();
 }
+
+/// Emit `cfg` values that can be used for conditional compilation based on the network backend.
+/// TODO: Remove this function once systemd-networkd integration development is finished
+fn emit_net_backend_cfgs() {
+    let net_backend = if Some("1".to_string()) == std::env::var(NET_BACKEND_OVERRIDE_ENV).ok() {
+        NET_BACKEND_OVERRIDE
+    } else {
+        DEFAULT_NET_BACKEND
```

Comment:
very much a nit: our other `cfg!` compile guards use lowercase, kebab-case strings, e.g. `aws-k8s` etc. So these screaming snake cases might look a little out of place at their usage sites.

Reply:
The actual cfgs emitted will be: `wicked` and `systemd-networkd`.  These screaming snake case constants are set to those static strings. :)

Reply:
ah i see :+1: